ORBIT/manager.py

```diff
@@ -1015,6 +1200,33 @@ def turbine_capex_per_kw(self):
         _capex = self.config.get("turbine_capex", None)
         return _capex
 
+    @property
+    def total_capex(self):
+        """
+        Returns total BOS CAPEX including commissioning and decommissioning.
+        """
+
+        return (
+            self.bos_capex
+            + self.turbine_capex
+            + self.commissioning
+            + self.decommissioning
+        )
+
+    @property
+    def total_capex_per_kw(self):
```

Comment:
Does this method (and total_capex) still work if the user runs without a cash flow model?  How does the user flag this - set discount rate to zero?

Reply:
Yeah this will calculate regardless the same regardless of if/how the cash flow is configured. Is that what you were thinking? I figured this was just a sum of all capex line items.

Reply:
oh i see - this is totally independent of the npv/cash flow model.  great.